component/admin/views/webservices/tmpl/default.php

```diff
@@ -322,7 +322,7 @@ class="btn btn-success"
 							</div>
 							<div class="controls">
 								<button
-									class="btn btn-success"
+									class="btn btn-success lc-install_all_webservices"
```

Comment:
is based on this best practice http://phptest.club/t/codeception-tips-tricks/14/8